src/index.js

```diff
@@ -101,10 +101,16 @@ module.exports = {
     const binDest = path.join(binDir, options.name)
     options.logger(`Symlinking binary from ${binSrc} to ${binDest}`)
 
-    return fs.ensureDir(binDir, '0755')
-      .catch(wrapError('creating binary path'))
-      .then(() => fs.symlink(binSrc, binDest, 'file'))
-      .catch(wrapError('creating binary file'))
+    const bundledBin = path.join(options.src, options.bin)
+
+    return fs.pathExists(bundledBin)
+      .then(exists => {
+        if (!exists) {
+          throw new Error(`could not find the Electron app binary at "${bundledBin}". You may need to set the "bin" option to the correct binary.`)
```

Comment:
I don't think that we should encourage Linux users to use `options.bin`. That option is there in case you have multiple binaries or a custom script used to launch the app. The majority of the time this error is thrown because of `electron-packager` creating an executable which doesn't follow Linux naming conventions.
Either add a warning to the current message saying that users need to check the binary follows Linux naming conventions, or just tell them to use `executableName` option from `electron-packager`, or just don't say anything and just tell them that we couldn't find the binary. Anyway, we don't want to promote the use of names that don't follow naming conventions.

Reply:
I mistyped "bin".